src/benchmark_runner.cc

```diff
@@ -291,7 +291,7 @@ void BenchmarkRunner::DoOneRepetition() {
                                          has_explicit_iteration_count ||
                                          ShouldReportIterationResults(i);
 
-    b.Tearndown();
+    b.Teardown();
```

Comment:
nit: this could come right after the call to `DoNIterations` so it's bracketed by `Setup` and `Teardown`.

Reply:
done.